examples/measurements.py

```diff
@@ -0,0 +1,136 @@
+
+"""Example OpenHTF test demonstrating use of measurements.
+
+Run with (your virtualenv must be activated first):
+
+  python measurements.py
+
+Afterwards, check out the output in measurements.json.  If you open both this
+example test and that output file and compare them, you should be able to see
+where measurement values end up in the output and what the corresponding code
+looks like that sets them.
+
+TODO(someone): Write these examples.
+For more complex topics, see the validators.py and dimensions.py examples.
+
+For a simpler example, see the hello_world.py example.  If the output of this
+test is confusing, start with the hello_world.py output and compare it to this
+test's output.
+
+Some constraints on measurements:
+
+  - Measurement names must be valid python variable names.  This is mostly for
+    sanity, but also ensures you can access them via attribute access in phases.
+    This applies *after* any WithArgs() substitution (not covered in this
+    tutorial, see the phases.py example for more details).
+
+  - You cannot declare the same measurement name multiple times on the same
+    phase.  Technically, you *can* declare the same measurement on multiple
+    phases; measurements are attached to a specific phase in the output.  This
+    isn't recommended, though, because it makes it difficult to flatten a test's
+    measurements, which some output formats require.
+"""
+
+# Import this output mechanism as it's the specific one we want to use.
+from openhtf.io.output import json_factory
+
+# You won't normally need to import this, see validators.py example for
+# more details.  It's used for the inline measurement declaration example
+# below, but normally you'll only import it when you want to define custom
+# measurement validators.
+from openhtf.util import validators
+
+# Import a handful of useful names.  If you're worried about polluting
+# your namespace, you can manually import just the things you want, this
+# is just a convenience.  See names.py for an exhaustive list.
+from openhtf.names import *
+
+
+# Simple example of measurement use, similar to hello_world.py usage.
+@measures(Measurement('hello_world_measurement'))
+def HelloPhase(test):
+  test.measurements.hello_world_measurement = 'Hello!'
+
+
+# An alternative simpler syntax that creates the Measurement for you.
+@measures('hello_again_measurement')
+def AgainPhase(test):
+  test.measurements.hello_again_measurement = 'Again!'
+
+
+# Multiple measurements can be specified in a single decorator, using either of
+# the above syntaxes.  Technically, these syntaxes can be mixed and matched, but
+# as a matter of convention you should always use one or the other within a
+# single decorator call.  You'll also note that you can stack multiple
+# decorations on a single phase.  This is useful if you have a handful of simple
+# measurements, and then one or two with more complex declarations (see below).
+@measures('first_measurement', 'second_measurement')
```

Comment:
btw, measurements can now have spaces and still be easily set (`setattr()` works, but for constants, ugh), so you could put a space in here to show that off (or another phase since these comments are kinda long already)